### PR TITLE
Exposed root auth failure messages

### PR DESCRIPTION
**Pull request by dimas-b**

### CODE REVIEW

**Review by dougwettlaufer — COMMENTED** [review]
Looks good but would suggest we do it on the other two as well for consistency

File: persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java [hunk 1]
```diff
@@ -332,8 +332,9 @@ private void authorizeAuthenticationStatement(
         authorization.authorizeRoleManagement(
             authenticationSubject, role, grantee, scope, SourceAPI.CQL);
       } catch (io.stargate.auth.UnauthorizedException e) {
+        logger.debug("Unauthorized statement: " + scope, e);
         throw new UnauthorizedException(
-            String.format("Missing correct permission on role %s", role));
+            String.format("Missing correct permission on role %s: %s", role, e.getMessage()));
```
Comment (tomekl007) [inline persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java:337]: Isn't there any IT/unit-test that validates the exception message? I don't see any test changes in the stargate.

**Review by tomekl007 — APPROVED** [review]

File: persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java (hunk 1 above)
Comment (dimas-b) [reply persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateQueryHandler.java:337]: unit tests added

**Comment by dimas-b** [comment]
Added similar changes to C* 3.11 and 4.0 persistence modules, unit tests and rebased to the latest `master`

**Pull request merged**